mne/evoked.py

```diff
@@ -1436,6 +1436,66 @@ def _write_evokeds(fname, evoked, check=True):
         end_file(fid)
 
 
+@fill_doc
+def export_evokeds(fname, evoked, fmt='auto', **kwargs):
+    """Export evoked dataset to external formats.
+
+    This function is a wrapper for format-specific export functions. The export
+    function is selected based on the inferred file format. All arguments are
+    passed to the respective export function.
+
+    Supported formats
+        MFF (mff, uses `mne.io.egi.egimff.export_evokeds_to_mff`)
```

Comment:
I'm not sure how to correctly reference this function. Would I add it to the file i/o python reference?

Reply:
I would suggest to expose it as `mne.io.egi.export_evokeds_to_mff`

I would insert section in the API page just before `Creating data objects from arrays` which I would call `Exporting to other file formats`

ok for you? thx

Reply:
As we have decided in the past that `mne.io` is for *raw* I/O only, I'm inclined to make it `mne.evokeds.export_evokeds_mff`